sample/AirshipSample/app.js

```diff
@@ -260,42 +270,50 @@ export default class AirshipSample extends Component {
     }
 
     return (
-      <View style={styles.centeredContainer}>
-        <Image
-          style={{width: 300, height: 38, marginTop:50}}
-          source={require('./img/urban-airship-sidebyside.png')}
-        />
-        <View style={{height:75}}>
-        </View>
-        <EnablePushCell
-          notificationsEnabled={this.state.notificationsEnabled}
-          handleNotificationsEnabled={this.handleNotificationsEnabled}
-        />
-        <View>
-          {channelcell}
-        </View>
-        <NamedUserManagerCell
-          namedUserText={this.state.namedUserText}
-          handleNamedUserSet={this.handleNamedUserSet}
-          handleUpdateNamedUserText={this.handleUpdateNamedUserText}
-          handleRenderNamedUser={this.handleRenderNamedUser}
-        />
-        <TagsManagerCell
-          tagText={this.state.tagText}
-          tagsDS={this.state.tagsDS}
-          handleTagAdd={this.handleTagAdd}
-          handleTagRemove={this.handleTagRemove}
-          handleUpdateTagText={this.handleUpdateTagText}
-        />
-        <EnableLocationCell
-          locationEnabled={this.state.locationEnabled}
-          handleLocationEnabled={this.handleLocationEnabled}
-        />
-        <Text style={styles.instructions}>
-          Press Cmd+R to reload,{'\n'}
-          Cmd+D or shake for dev menu
-        </Text>
-      </View>
+      <View style={styles.backgroundContainer}>
```

Comment:
This is me adding the view hierarchy into a scroll view and adding the message button functionality